### PR TITLE
App updates for user-feedback analysis

### PR DESCRIPTION
**Pull request by mudrunka**
Does 3 standard analysis and then merges them together.

### CODE REVIEW

File: src/analysis_app.py [hunk 1]
```diff
@@ -221,45 +218,87 @@ def analyze(self, row_stream):
...
+                neg_analysis = analysis_by_type['neg']
+                self.proc_entities(neg_analysis['entities'], 'neg')
+                self.copy_analysis(neg_analysis, doc_analysis)
+            yield doc_analysis
+
+    def proc_entities(self, entities, doc_type):
+        feedback_entities = [e for e in entities if e['type'] in self.params.feedback_entities]
+        for ent in feedback_entities:
+            polarity = ent.get('sentiment', {}).get('polarity', 0)
+            if doc_type == 'pos' or (doc_type == 'txt' and polarity > 0):
```
Comment (vstejskal) [inline src/analysis_app.py]: I think we should treat neutral polarity as positive. Hence, I'd use `(doc_type == 'txt' and polarity >= 0)`.

**Review by vstejskal — APPROVED** [review]

**Comment by vstejskal** [comment]
I wonder where the UI definition is? In KBC the app's UI says 'section with positive comments' for both the positive and negative columns. I think that it's just the label in UI, but I couldn't find where it is defined.

**Comment by mudrunka** [comment]
The UI is defined using a JSON-schema in https://components.keboola.com/~/components/geneea.feedback-analysis. There may be some typos there.

File: src/analysis_app.py (hunk 1 above)
Comment (mudrunka) [reply src/analysis_app.py]: 2791c13

**Pull request merged**